### PR TITLE
Add font weight heavy variable

### LINKED ISSUE #29778: [Issue] Add font weight heavy variable
This issue is automatically created based on existing pull request: magento/magento2#29773: Add font weight heavy variable

### Preconditions (*)
1. Magento 2.4

### Steps to reproduce (*)
Small additional to core lib. Add more font weight for reuse

### Expected result (*)
Adding more font weight for reuse

### PR DESCRIPTION
**Pull request by mrtuvn**
### Description (*)

Small additional to core lib. Add more font weight for reuse

### Fixed Issues (if relevant)

1. Fixes magento/magento2#<issue_number>

### Manual testing scenarios (*)

1. ...
2. ...

### Contribution checklist (*)
 - [ ] Pull request has a meaningful description of its purpose
 - [ ] All commits are accompanied by meaningful commit messages
 - [ ] All new or changed code is covered with unit/integration tests (if applicable)
 - [ ] All automated tests passed successfully (all builds are green)

### Resolved issues:
1. [x] resolves magento/magento2#29778: Add font weight heavy variable

### CODE REVIEW

**Comment by ihor-sviziev** [comment]
@mrtuvn maybe it's good idea to add all common weight names? https://developer.mozilla.org/uk/docs/Web/CSS/font-weight#Common_weight_name_mapping

**Comment by mrtuvn** [comment]
good point!. I have updated in my PR

File: lib/web/css/source/lib/variables/_typography.less [hunk 1]
```diff
@@ -39,11 +39,15 @@
 @font-size__xs: floor(.75 * @font-size__base); // 11px
 
 //  Weights
+@font-weight__hairline: 100;
+@font-weight__thin: 200;
```
Comment (ihor-sviziev) [inline lib/web/css/source/lib/variables/_typography.less]: from the table: 

Value | Common weight name
-- | --
200 | Extra Light (Ultra Light)



```suggestion
@font-weight__extralight: 200;
```

**Review by ihor-sviziev — CHANGES_REQUESTED** [review]

**Review by ihor-sviziev — APPROVED** [review]

**Comment by ihor-sviziev** [comment]
@magento run all tests

**Comment by magento-engcom-team** [comment]
Hi @ihor-sviziev, thank you for the review.
[ENGCOM-8085](https://jira.corp.magento.com/browse/ENGCOM-8085) has been created to process this Pull Request

**Comment by sidolov** [comment]
@magento create issue

**Review by ptylek — APPROVED** [review]

**Comment by magento-engcom-team** [comment]
Hi @ptylek, thank you for the review.
[ENGCOM-8085](https://jira.corp.magento.com/browse/ENGCOM-8085) has been created to process this Pull Request

**Review by ihor-sviziev — APPROVED** [review]

**Comment by magento-engcom-team** [comment]
Hi @ihor-sviziev, thank you for the review.
[ENGCOM-8085](https://jira.corp.magento.com/browse/ENGCOM-8085) has been created to process this Pull Request

**Comment by engcom-Alfa** [comment]
@magento create issue

**Comment by engcom-Alfa** [comment]
:heavy_check_mark: QA Passed

possible to use new font-weight variable values

**Pull request merged**